nuget/helpers/lib/NuGetUpdater/NuGetUpdater.Core/Run/ApiModel/DependencyGroup.cs

```diff
@@ -1,8 +1,68 @@
+using System.Collections.Immutable;
+using System.IO.Enumeration;
+
 namespace NuGetUpdater.Core.Run.ApiModel;
 
 public record DependencyGroup
 {
     public required string Name { get; init; }
     public string? AppliesTo { get; init; }
+
+    // TODO: make more strongly typed, but currently this seems to be:
```

Comment:
Got an issue/work item tracking this?

Reply:
https://github.com/dependabot/dependabot-core/issues/12374